src/views/GameDetailView.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useGameLibraryStore } from '../stores/GameLibraryStore.js'; 
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
+
+const gameLibraryStore = useGameLibraryStore();
+
+if(!gameLibraryStore.count) {
+  await gameLibraryStore.getGameDetails(route.params.id);
+}
+</script>
+
+<template>
+  <main class="game-detail-view">
+    <h1>Game Details</h1>
+
+    <h2>{{ gameLibraryStore.gameDetails.name }}</h2>
+
+    <img :src="gameLibraryStore.gameDetails.imageUrl" loading="lazy" />
+
+    <p class="game-description">{{ gameLibraryStore.gameDetails.description }}</p>
+
+    <ul class="game-info-list">
+      <li>
+        <b>min players: </b>{{ gameLibraryStore.gameDetails.minPlayers }}
+      </li>
+      <li>
+        <b>max players: </b>{{ gameLibraryStore.gameDetails.maxPlayers }}
+      </li>
+      <li>
+        <b>playing time: </b>{{ gameLibraryStore.gameDetails.playingTime }}
+      </li>
+    </ul>
+
+    <div class="category_container">
+      <span class="category" v-for="(category) in gameLibraryStore.gameDetails.categories" :key="index">
```

Comment:
update to the v-for to index a definition for `index`

`v-for="(category, index) in gameLibraryStore.gameDetails.categories`

The index in `:key="index"` isn't defined without it